main.go

```diff
@@ -115,17 +131,84 @@ func ProcessIncomingConnection(conn net.Conn) {
 
 		if decoder.Method == "icinga::Hello" {
 			log.Printf("SERVER: JsonRpcConnection: Received %q from the client\n", decoder.Method)
-		}
 
-		err = base.WriteNetStringToStream(conn, mes)
+			var encoder Encoder
+			encoder = Encoder{
+				JSONRPC: "2.0",
+				Method: "config::Update",
+				Params: Updates{
+					Update: map[string]ConfigInfo{
+						*zone: {},
+					},
+					Update_v2: map[string]ConfigInfo{
+						*zone: {},
+					},
+					Checksums: map[string]ConfigInfo{
+						*zone: {},
+					},
+				},
+			}
 
-		log.Printf("SERVER: Connection: Sending %d bytes message back", len(mes))
+			r := math.Intn(*files)
+			for i := 0; i < r; i++ {
+				file, err := GenerateFile("conf")
 
-		if err != nil {
-			log.Printf("SERVER: Connection: Sending data to the client Error: %s", err)
-			break
+				if err != nil {
+					log.Fatalf("SERVER: JsonRpcConnection: Error while generating new config files: %s", err)
+				}
+
+				encoder.Params.Update[*zone][file] = file
+
+				path, err := GenerateFile("conf")
+
+				if err != nil {
+					log.Fatalf("SERVER: JsonRpcConnection: Error while generating new config files: %s", err)
+				}
+
+				encoder.Params.Update_v2[*zone][path] = path
+
+				random, err := GenerateFile("conf")
+
+				if err != nil {
+					log.Fatalf("SERVER: JsonRpcConnection: Error while generating new config files: %s", err)
+				}
+
+				encoder.Params.Checksums[*zone][random] = random
```

Comment:
* According to #7 the files shall be empty
* The file names shall be the same for Update, Update_v2 and Checksums